SampleProject/VOKCoreDataManagerTests/VOKManagedObjectAdditions.m

```diff
@@ -156,17 +154,17 @@ - (void)testRecordInsertionBackgroundThreadConvenienceReturningManagedObjectsArr
                                  @"numberOfHats" : @1,
                                  },
                              ];
-
+    
     [VOKThing vok_addWithArrayInBackground:importArray
-                               completion:^(NSArray *arrayOfManagedObjects) {
```

Comment:
Probably a future thing and not for this PR, but could `vok_addWithArrayInBackground:completion:` be set up so that the completion parameter was `NSArray<instancetype> *`? (I'm pretty sure `instancetype` can't be used in that way, but maybe achieve that effect through some other means?)